### DISCUSSION TITLE
停止接受【开屏广告以外】的规则及其任意更新及其BUG修复

### DISCUSSION THREAD

**Discussion by lisonge** [gkd-kit/gkd#507] [General]
# 首先在此感谢各位开发者的无私贡献

是各位开发者的无私贡献才使得 GKD 的订阅增加了很多有用的规则

---

但是本人精力有限，且本人从未(**也无意**)从 GKD 的开源工作中获取任何回报(这点三瓜两枣还顶不上我半天的工资)

大家都是白嫖怪(我也是)，也别说什么冠冕堂皇的话了，这点逼数我心里还是有的，务实一点

因此经过慎重考虑(懒得审核了,不想被大家白嫖了)，决定停止接受【开屏广告以外】的规则及其任意更新及其BUG修复

---

此外下个版本的 GKD 默认将不附带任何规则，在 GKD 里添加其它规则也会显示未验证的来源的弹窗提示，实现规则与应用本体完全分离

同时 GKD 的文档 <https://github.com/gkd-kit/docs> 也将逐步完善，尽可能降低编写规则的上手难度

---

后续追加

本人审核精力有限，所有规则均暂时停止维护, 你可以自行编写规则, 导入本地订阅使用

**Comment by lisonge** [comment]
手动 at 所有贡献者，如果这条消息打扰到您，请您 忽略/评论/邮箱 告知我关闭对您的 at

@Snoopy1866 @lisonge @jiuqianyuan @github-actions[bot] @walklty @aoguai @xiaowonet @adproqwq @CMBill @yanyu2023 @dlh996 @Otis0227 @shanlanCoding @BreezyOrigin @Sec-ant @zeXtreme @superHao2000 @longm21 @xe5700 @JammyLin @Nebulafalls @OldCoding @Ran0929 @YaChengMu @shjdgwj @zhyx490991014 @zhangyusheng211 @lliioollcn @lyc452 @M1saka10010 @xuanqb @sishuikaki @sherylynn @ZhReimu @bleach2012 @javabcde @kyrinyoga @zhuzhuani @wjf0214 @uuiid @MoeGrid @g1fts @small-ora @China-Uncle @doio @Crty @someone120 @koo6666 @h7221836 @maxchang3 @Schweik7 @weirongchuan @pengxinwen @Zonezzc @alzuobaba @guanyintu @marryton007 @foxlesbiao @lsxuan12138 @luoyan87 @ppengit @w4616 @shiquda @tiygu @xhemj @xzhlksh @yakun4566 @yangqiwen1024 @ylxz95 @ysjsgzq @zhcode-fun @yunyuyuan

**Comment by Crty** [comment]
你好 邮件已经收到

**Comment by aoguai** [comment]
😭变相停止维护了吗

---
🙏🏻感谢大佬开发的APP，如果希望自己fork一个分支维护自己的规则的话，我该如何生成最终的json文件供导入本地使用呢？（本人在偏前端属于小白，TypeScript 也不是很懂）

**Reply by adproqwq** [reply]
https://github.com/Adpro-Team/GKD_subscription/commit/f9b5aea3b80e04b2632170c7f151dc5df0b71711
https://github.com/Adpro-Team/GKD_subscription/commit/dcc495a2395bc55efd7c474302f8de343d305427
---
可以参考我改的

**Reply by aoguai** [reply]
> ## [Adpro-Team/GKD_subscription@f9b5aea](https://github.com/Adpro-Team/GKD_subscription/commit/f9b5aea3b80e04b2632170c7f151dc5df0b71711)
> [Adpro-Team/GKD_subscription@dcc495a](https://github.com/Adpro-Team/GKD_subscription/commit/dcc495a2395bc55efd7c474302f8de343d305427)
> 可以参考我改的

感谢，我研究一下

**Comment by lisonge** [comment]
上述通知已更新，麻烦大家再次查看

**Comment by no-swing** [comment]
感谢，把自己改过规则直接放进本地也行

**Comment by OUISEDRG** [comment]
才知道这个仓库，太可惜了

**Comment by dlh996** [comment]
一直想自己维护一个自用的版本，奈何越搞越觉得头大。所以对大佬的决定，深表理解。

**Comment by uuiid** [comment]
理解,  理解,

**Comment by huasuxu** [comment]
感谢(^🙏^)新年快乐！

**Comment by ijustsomebody** [comment]
理解，无论如何，新年快乐

**Comment by 0430zhang** [comment]
加油吧各位

**Comment by Ym-wei** [comment]
理解, 大佬新年快乐
